pkg/descheduler/evictions/evictions.go

```diff
@@ -137,6 +145,19 @@ func (pe *PodEvictor) EvictPod(ctx context.Context, pod *v1.Pod, opts EvictOptio
 		return false
 	}
 
+	podOwnerRefKey := utils.GetPodOwnerRefKey(pod)
+	klog.V(1).Infof("Pod's OwnerReference Replicas is %v", pe.PodReplicasCount[podOwnerRefKey[0]])
```

Comment:
`utils.GetPodOwnerRefKey` can return an empty list so `podOwnerRefKey[0]` will fail.

How should this handle bare pods?